src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml

```diff
@@ -93,7 +93,7 @@
     </div>
 </div>
 <div class="row" data-review-id="@Model.Review.ReviewId" data-revision-id="@Model.Revision.RevisionId">
-    <div class="col-lg-2 col">
+    <div id="review-left">

```

Comment:
Make sure the view is still responsible without those classes:
![image](https://user-images.githubusercontent.com/1697911/83457459-267a3380-a416-11ea-84b0-79d5137b0478.png)

Reply:
It's not after this change, but I should be able to put the old styles back and this should still work since the explicitly calculated element widths should override the CSS's widths.